cmd/snap-confine/mount-support-nvidia.c

```diff
@@ -119,6 +119,39 @@ static const char *nvidia_globs[] = {
 	"libnvoptix.so*",
 	"tls/libnvidia-tls.so*",
 	"vdpau/libvdpau_nvidia.so*",
+
+	// additional libraries for Tegra
```

Comment:
Can this comment please list a URL or similar that describes these libraries?

Reply:
yeah, that would be great, but there is no such thing. also: this should be fixed by nvidia.

Reply:
I've added links to some scraps of documentation I could find. Unfortunately, it's probably as good as it will ever get.